perfil/election/models.py

```diff
@@ -22,6 +22,23 @@ def __str__(self):
         return f'{self.candidate.civil_name} - {self.year} - {self.position}'
 
 
+class Donation(models.Model):
+    election = models.ForeignKey(Election, on_delete=models.CASCADE)
+    donator = models.CharField(max_length=250)
+    donator_id = models.CharField(max_length=14, null=True)
```

Comment:
all the CharField fields should not have `null=True` because we could both have empty string and None as a value. We should change it to blank=True

`Avoid using null on string-based fields such as CharField and TextField. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL.` on [this part of documentation](https://docs.djangoproject.com/en/2.0/ref/models/fields/)

Reply:
Fixed in e8af033